src/signals/incident/components/form/MapSelectors/Asset/Selector/WfsLayer/AssetLayer/AssetLayer.tsx

```diff
@@ -206,7 +210,10 @@ export const AssetLayer: FunctionComponent<DataLayerProps> = ({
               description,
               id,
               type: typeValue,
-              isReported: feature.properties.meldingstatus === 1,
+              isReported: Boolean(
+                feature.properties[reportedFeatureType.isReportedField] ===
+                  reportedFeatureType.isReportedValue
```

Comment:
`reportedFeatureType` can be `undefined`

Reply:
✅